README.md

```diff
@@ -27,6 +27,40 @@ Meeting Agenda templates (located in the `/docs` directory in this repo):
 - Meeting - Sprint Retrospective, Review, and Planning --> ./docs/meeting-sprint_retrospective_review_and_planning.docx
 - Meeting - Sprint Open Topic Session --> ./docs/meeting-sprint_open_topic_session.docx
 
+## Getting Started
+### Installation
+
+#### 1. Clone repository
+```
+git clone https://github.com/chingu-voyages/v49-tier3-team-29.git
+```
+#### 2. Install npm packages both in "client" and "backend" directory.
+```
+npm install
+```
+#### 3. Create .env file both in "client" and "backend" directory.
+#### client > .env
+```
+# Demo User Credentials
+VITE_DEMO_USERNAME=demo_user
+VITE_DEMO_PASSWORD=demo4929
```

Comment:
Would VITE_DEMO_PASSWORD have to match up with the DEMOUSER_PASSWORD on the backend?

Reply:
Yes, they should be identical. Thanks for reminding me. I will add the information.